src/Observable.php

```diff
@@ -304,15 +372,17 @@ public static function defer(callable $factory)
      *
      * @param $start
      * @param $count
+     * @param SchedulerInterface $scheduler
      * @return RangeObservable
+     * @throws \InvalidArgumentException
      *
      * @demo range/range.php
      * @operator
      * @reactivex range
      */
-    public static function range($start, $count, SchedulerInterface $scheduler = null)
+    public static function range(int $start, int $count, SchedulerInterface $scheduler = null): RangeObservable
```

Comment:
After looking at the implementation - I think we should leave this as `int` - See below